Packs/qualys/Integrations/integration-qualys.yml

```diff
@@ -433,7 +433,8 @@ script:
         'vmScanAction': { ContextPath: undefined,
                         ContentPath: ['SIMPLE_RETURN.RESPONSE','.ITEM_LIST.ITEM'],
                          entityType: ''},
-        'vmScanFetch': '',
+        'vmScanFetch': { ContextPath: 'Qualys.VM',
```

Comment:
No DT is required here?

Reply:
No, i have checked it. The output of that command is different from other commands.

Reply:
And what happens when you run the same command more than once? Then the context would have duplicates or not?

Reply:
No, It will override

Reply:
Change to DT